app/cafe.py

```diff
@@ -0,0 +1,19 @@
+import datetime
+from app.errors import (NotVaccinatedError, OutdatedVaccineError,
```

Comment:
After opening the parentheses, write all the arguments on a new line, including the first one, the closing bracket is also written on a new line